packages/utils/logger/lib/config/output-file-configuration.js

```diff
@@ -1,9 +1,13 @@
 'use strict';
 
 const { transports, format } = require('winston');
-const { LEVEL_LABEL, LEVELS } = require('./constants');
-const { prettyPrint } = require('./formats');
+const { LEVEL_LABEL, LEVELS } = require('../constants');
+const { prettyPrint } = require('../formats');
 
+/**
+ * This will remove the chalk color codes from the message provided.
+ * It's used to log plain text in the log file
+ */
 const getPlainText = format.printf(({ message }) => {
```

Comment:
We could probably extract this format to the format directories, but we can do it later.